docs/reference/pip_install.rst

```diff
@@ -89,6 +93,25 @@ Some Examples:
   Don't use single or double quotes in a ``requirements.txt`` file.
 
 
+.. _`Per-requirement Overrides`:
+
+Per-requirement Overrides
++++++++++++++++++++++++++
+
+It is possible to set ``--install-options`` and ``--global-options``
+for each requirement in the requirements file:
+
+ ::
+
+    FooProject >= 1.2 --install-options="--prefix='/usr/local'" \
+                      --global-options="--no-user-cfg"
+
+The above translates roughly into running FooProject's ``setup.py``
+script as:
+
+ ::
+
+    python setup.py --no-user-cfg install --prefix='/usr/local'
```

Comment:
Actually, it looks like ```-no-user-cfg``` isn't supported for Python 2.6, which is causing the build failures there.

It seems to me that this is a significant behaviour change (I have a ```$HOME/pydistutils.cfg``` file that points to some library directories where I store compiled libraries that some of my builds need) and I don't think it should just be slipped in as part of this change.